docs/mkdocshelptheme/README.md

```diff
@@ -0,0 +1,29 @@
+MkDocs related example to fix the permalink below the top bannel.
```

Comment:
mkdocshelptheme folder will be removed: checked with MkDocs team for a better solution